Collection settings (#202)
Macrometacorp/docs: docs/collections/_partials/_view-collection-settings.md

## View Collection Settings

1. Log in to your [Macrometa account](https://auth.paas.macrometa.io/).
1. Click **Collections**.

> 1. Click the collection that you want to view the settings of.

Reviewer: ```suggestion
1. Click the collection for which you want to view the settings.
```
